### DISCUSSION TITLE
Single item selected from a list - Description display

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#77] [General]
In my property schema I have defined a description for a property of type - "Single Item from Selected List".  I have assigned the property schema to a item template.

When I use the item template in the threat model where can I see the description field ?

If I am using a text field or boolean field I am able to see the description field as a tool tip on the diagram when my mouse hovers over the field .
![image](https://github.com/simonec73/threatsmanager/assets/31912418/e6f6197a-17fa-4c38-9e20-4a767765633d)

Is there any place I can see the description field for drop down field ("Single item from Selected List") ?

**Comment by simonec73** [comment]
Great catch! Version 2.2.3 will fix this issue. It is planned for end of June. Thank you.
